core/whistle-1.0.0/src/wh_cache.erl

```diff
@@ -143,6 +145,11 @@ start_link(Name, ExpirePeriod, Props) ->
                                    )
     end.
 
+-spec maybe_add_db_binding(wh_proplists()) -> wh_proplists().
+maybe_add_db_binding([[]]) -> [[]];
```

Comment:
I commened `maybe_add_db_binding([[]]) -> [[]];` restart whistle_apps
`|undefined|wh_cache:138 (<0.1070.0>) DEBUG Bindings: [{conf,[federate,{type,<<"database">>}]},{conf,[federate]}]`
Then i look to rabbitmq bindings (rabbitmqctl list_bindings). Its show me only one binding
`configuration   exchange        whistle_apps@dev.dinkor.net-<0.1071.0>-999f5196     queue   *.*.database.*  []`
no `*.*.*.*` bindings.

I think `[{conf,[federate,{type,<<"database">>}]},{conf,[federate]}]` is "optimzed" to `[{conf,[federate, {'type', <<"database">>}]}]`